Resources/Private/Templates/FrontendLogin/Login/Logout.html

```diff
@@ -0,0 +1,23 @@
+<html data-namespace-typo3-fluid="true" xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers" xmlns:mrc="http://typo3.org/ns/Hemmer/MrCore/ViewHelpers">
+    <f:form action="login" actionUri="{actionUri}" class="felogin" data="{action: 'sign-out'}" fieldNamePrefix="" hiddenFieldClassName="d-none">
+        <div class="d-none">
+            <f:form.hidden name="logintype" value="logout"/>
+            <f:form.hidden name="pid" value="{storagePid}"/>
+        </div>
+        <mrc:fal as="avatar" data="{user}" table="fe_users">
+            <f:if condition="{avatar.0}">
+                <f:then>
+                    <button aria-label="{f:translate(extensionName: 'MrFelogin', id: 'sign_out')}" class="page-icon nav-link" name="submit">
```

Comment:
`MrFeLogin` has certainly nothing to do with that extension nor any extension in the core...